1-mandatory/2-people-i-know.js

```diff
@@ -378,13 +378,17 @@ First, I want you to find all of my friends who are 35 or older.
 */
 
 let thirtyFiveOrOlder = [];
+thirtyFiveOrOlder = friends.filter((friend) => friend.age >= 35);
 
 /*
 3) Find the email address
 Next, I want you to find all of my friends who work for "POWERNET" and then store their emails in the array below
 */
 
 let powerNetEmails = [];
+powerNetEmails = friends
```

Comment:
map method returns array. so no need to initialize empty array in this case.

let powerNetEmails = friends...

Reply:
Thank you very much :pray:
Yes, you're right. I knew about it and I just try to not change their codes.